internal/storage/v2/clickhouse/tracestore/dbmodel/dbmodel.go

```diff
@@ -5,51 +5,43 @@ package dbmodel
 
 import "time"
 
-// Trace Domain model in Clickhouse.
-// This struct represents the schema for storing OTel pipeline Traces in Clickhouse.
-type Trace struct {
-	Resource Resource
-	Scope    Scope
-	Span     Span
-	Links    []Link
-	Events   []Event
-}
-
-type Resource struct {
-	Attributes AttributesGroup
-}
-
-type Scope struct {
-	Name       string
-	Version    string
-	Attributes AttributesGroup
-}
-
+// Span represents a single row in the ClickHouse `spans` table.
 type Span struct {
-	Timestamp     time.Time
-	TraceId       string
-	SpanId        string
-	ParentSpanId  string
-	TraceState    string
-	Name          string
-	Kind          string
-	Duration      time.Time
-	StatusCode    string
-	StatusMessage string
-	Attributes    AttributesGroup
+	// --- Span ---
```

Comment:
@yurishkuro There's a couple of fields missing from the OTLP type. Do we want to store these?
- Flags
- DroppedAttributesCount
- DroppedEventsCount
- DroppedLinksCount

Reply:
yes we can add these too, doesn't hurt.

Reply:
This PR is already quite large so I'll add these in a following PR